README.md

```diff
@@ -156,9 +156,9 @@ client.delete(key)
 
 ### Build information:
 
-* aerospike          - 3.12.1
-* aerospike-c-client - 4.1.5
-* ruby version       - 2.3.0
+* aerospike          - 3.14.1.4
+* aerospike-c-client - 4.1.6
+* ruby version       - 2.4.1
```

Comment:
can u upgrade to 2.4.2? It fixed a lot of important vulnerabilities https://www.ruby-lang.org/en/news/2017/09/14/ruby-2-4-2-released/